File: Database/Assembly_SRPG/SimpleQuestDropParam.cs
namespace SRPG
{
    using System;

    public class SimpleQuestDropParam
    {
        public string item_iname;
        public string[] questlist;

        public SimpleQuestDropParam()
        {
            base..ctor();
            return;
        }

        public bool Deserialize(JSON_SimpleQuestDropParam json)
        {
            this.item_iname = json.iname;
            this.questlist = json.questlist;
            return 1;
        }
    }
}

File: Database_V2/ParamFunctions/SimpleQuestDropParam.py
def SimpleQuestDropParam(json):
    this={}#SimpleQuestDropParamjson)
    if 'iname' in json:
        this['item_iname'] = json['iname']
    if 'questlist' in json:
        this['questlist'] = json['questlist']
    return this
